packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,33 +1,75 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
 import logoImage from "./../../assets/blood-bank-zichron-logo.svg";
+import popupImage from "../../assets/popup_resetpass.png";
```

Comment:
Please use SVG and not PNG.

SVG stays sharp even when zooming in

Reply:
done